devtools/gradle/gradle-extension-plugin/gradle.properties

```diff
@@ -1 +1 @@
-jacksonVersion = 2.12.4
\ No newline at end of file
+jacksonVersion = 2.12.5
```

Comment:
This one could be removed then too.

Reply:
I added a platform to the plugin and remove the version